### PR TITLE
[CR-2088] Add frequent payments option

### PR DESCRIPTION
**Pull request by antonioliv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,20 @@ <h1>easypay Checkout SDK Demo</h1>
     <div class="options">
       <div class="section">
         <h3>Payment Type:</h3>
+        <div class="option">
+          <label for="switch">Single</label>
+          <div id="switch" class="switch" onclick="toggleOption('single')">
```
Comment (Arlanthir) [inline index.html:16]: Can you show a screenshot please? I wonder if maybe radio buttons or a select would be better than the switches that turn each other off?

Beware the multiple equal ids.

Comment (antonioliv) [reply index.html:16]: https://www.loom.com/share/d6653e5cb4f64c7bae88b7b924673b2c

Comment (antonioliv) [reply index.html:16]: Captura de ecrã 2022-12-23, às 14 58 00

**Review by Arlanthir — APPROVED** [review]
LGTM!

**Pull request merged**